docker-app/qfieldcloud/urls.py

```diff
@@ -61,4 +61,6 @@
     path("auth/", include("rest_framework.urls")),
     path("accounts/", include("allauth.urls")),
     re_path(r"^invitations/", include("invitations.urls", namespace="invitations")),
+    # We don't use those for now, but leaving that here until we're sure
+    # path("inbox/notifications/", include(notifications.urls, namespace="notifications")),
```

Comment:
What is this?

Reply:
It's some preset views to manage notifications (provided by the notifications app), see https://github.com/django-notifications/django-notifications/blob/master/notifications/urls.py

I think it's more effort integrating those views in our view system than reimplementing... I'll just remove that comment

Reply:
Ah, I see. Yeah, I guess removing the comments would be better.